RELEASE-NOTES.md

```diff
@@ -1,5 +1,52 @@
 # Prefect Release Notes
 
+## Release 2.16.5
+
+### Exciting New Features 🎉
+- The flow run graph UI enhancements for visualizing state changes and artifacts added in 2.16.4 are now enabled by default.
+
+### Enhancements
+- Render artifacts and states on the flow run graph by default. — https://github.com/PrefectHQ/prefect/pull/12300
+- Keep artifacts file in prefect-client — https://github.com/PrefectHQ/prefect/pull/12316
+- Add a select all flow runs checkbox to the flow runs page. — https://github.com/PrefectHQ/prefect/pull/12356
+- remove feature flagging around enhanced-deployment-experiment — https://github.com/PrefectHQ/prefect/pull/12360
+- Feature : #11773 UI: Add checkboxes for runs for an individual flow to allow multi-selection/-deletion — https://github.com/PrefectHQ/prefect/pull/12285
+- Retry 500s to make flow and task runs more resilient — https://github.com/PrefectHQ/prefect/pull/12084
+- Add a capability to verify ssl certificates in Prefect CLI — https://github.com/PrefectHQ/prefect/pull/11771
+- Add prefect task-runs to cli — https://github.com/PrefectHQ/prefect/pull/12307
+- [background tasks] emit pending event — https://github.com/PrefectHQ/prefect/pull/12352
+- Update `CsrfTokenApi` to retry failed requests due to invalid tokens — https://github.com/PrefectHQ/prefect/pull/12373
+
+### Fixes
+- Refactor logic to set `task_key` for background tasks — https://github.com/PrefectHQ/prefect/pull/12337
+- Correct a memory leak with the outbound task run websockets — https://github.com/PrefectHQ/prefect/pull/12346
+- fix(task/flow): correct type hint for possibly async `@flow` hooks — https://github.com/PrefectHQ/prefect/pull/12231
```

Comment:
maybe more like 'correctly type hint in flow run state change hooks' or something